flake.nix

```diff
@@ -153,7 +153,7 @@
       }
       (inputs.tullia.fromStd {
         actions = inputs.std.harvest inputs.self [ "cloud" "actions" ];
-        tasks = inputs.std.harvest inputs.self [ "cloud" "pipelines" ];
+        tasks = inputs.std.harvest inputs.self [ "automation" "pipelines" ];
```

Comment:
What's the rationale? Maybe we need to update the comments above explaining the purposes of the cell blocks?

Reply:
Tullia tasks are in the automation cell in every other stdized repo and [this](https://github.com/input-output-hk/plutus/blob/3a7af5130920661567de0de8fe73fa4d22276201/nix/cells/cloud/pipelines.nix#L9-L10) comment indicated that it was only in the cloud cell because it had to be, but that condition has been fixed.

The logic behind that is that tullia tasks can be run locally (not in the cloud), so they belong in the automation cell. Cicero actions can only run in the cloud (and happen to run tullia tasks) so they belong in the cloud cell.